include/mata/rrt.hh

```diff
@@ -25,6 +25,14 @@
 
 namespace Mata
 {
+/**
+ * Restricted Register Transducer (over finite words) including structures and algorithms.
+ *
+ * These are restricted version of a (nondeterministic) register automaton that still has some
+ * decidable properties and closure under operations.
+ *
+ * TODO: Quite obsolete, should we remove this?
```

Comment:
We should discuss it shortly on meeting.

Reply:
I believe there are plans to expand upon this in the future. It should not be obsolete. We will take a look at this in our meeting as well.

Reply:
We have many plans, the question is whether to keep it in master or copy it to some branch to archive it

Reply:
That is good point, we could simply delete it from index, then create isolate branch, commit it and then push it as branch + PR.

Reply:
This conversation has its own issue now: #217. I consider this conversation here to be resolved.